services/api-server/src/simcore_service_api_server/api/dependencies/database.py

```diff
@@ -1,5 +1,5 @@
 import logging
-from typing import AsyncGenerator, Callable
```

Comment:
Question for my learning: Why this change?

Reply:
ruff suggestion. python is moving these classes to collections in later version